easybuild/easyconfigs/n/NBO/NBO-7.0.10-GCC-10.2.0-i4.eb

```diff
@@ -0,0 +1,56 @@
+# this recipe is configured for use with Gaussian G16, revision A03 or later
+
+easyblock = 'MakeCp'
+
+name = 'NBO'
+version = '7.0.10'
+versionsuffix = '-i4'
+
+homepage = 'http://nbo.chem.wisc.edu/'
+description = """ The Natural Bond Orbital (NBO) program is a discovery tool
+for chemical insights from complex wavefunctions. """
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+toolchainopts = {'pic': True, 'i8': True}  # remove 'i8' if Gaussian was compiled with 32-bit integers
```

Comment:
As `versionsuffix = '-i4'` and `local_intlength = 'i4'` should the `'i8': True` be removed?

Reply:
Yes, you are right. I checked the log-file and somehow that actually did not show up at all but it is more consistent to have it removed. Thanks.